src/program/lwaftr/virt/lwaftrctl

```diff
@@ -4,99 +4,90 @@
 #
 # The scripts enables several commands:
 #
-# * lwaftrctl snabbnfv start|stop. Starts or stops snabbnfv.
-# * lwaftrctl vm       start|stop. Starts or stops virtual machine
-# * lwaftrctl lwaftr   start|stop. Starts or stops lwaftr inside virtual machine.
+# * lwaftrctl start|stop snabbnfv   Starts or stops snabbnfv.
+# * lwaftrctl start|stop vm         Starts or stops virtual machine
```

Comment:
Probably I did because I use "systemctl" often which has syntax "systemctl start|stop|restart service". I also thought this was a good idea since it follows Western languages SVO  (syntax-verb-object) grammar. I like it more now but perhaps is because I'm used to "systemctl".